docs/creating_content_basic.md

```diff
@@ -105,10 +105,9 @@ You can select a SiteAccess to preview by using a drop-down in the preview scree
 
 To keep your content organized and easy to find, you can add taxonomy entries to a Content item while creating or editing it.
 
-1. Go to **Taxonomy entries** section.
+1. Go to section with tag field.
 1. Click the **Select Taxonomy Entries** button.
-1. In the pop-up window, select tags you want to add.
-You can add as many tags as you like.
+1. In the pop-up window, select tag category and tags you want to add.
```

Comment:
If taxonomy type is defined at Content Type level, then you don't select category here, but only tags?

Reply:
Yes, only tags, categories are selected in the Content Type level, in the Field type settings.